doc/7/core-classes/observer/constructor/index.md

```diff
@@ -0,0 +1,38 @@
+---
+code: true
+type: page
+title: constructor
+description: Creates a new Observer object
+order: 50
+---
+
+# Constructor
+
+Use this constructor to create a new Observer instance.
+
+## Arguments
+
+```js
+Observer(sdk, [options]);
+```
+
+<br/>
+
+| Argument  | Type              | Description                    |
+|-----------|-------------------|--------------------------------|
+| `sdk`     | <pre>Kuzzle</pre> | SDK instanciated and connected |
+| `options` | <pre>object</pre> | Additional options             |
+
+### options
+
+Observer options.
+
+| Property       | Type<br/>(default)             | Description                                                  |
+|----------------|--------------------------------|--------------------------------------------------------------|
+| `pullingDelay` | <pre>number</pre><br/>(`5000`) | Refresh delay in ms when the SDK is using the HTTP protocol. |
```

Comment:
I'm not sure about that because there is the HTTP long Polling mechanism  but this is not. The Observer is pulling the data every 5 sec but maybe I'm wrong, I'm not sure to know all meanings of "poll"

Reply:
I agree @xbill82 (https://stackoverflow.com/a/2761232/10884892)

Reply:
HTTP Long polling and the Observer "pulling / polling" are actually two different polling mechanism. Because, from whatever point you look at it, the client is periodically polling the server for data.
So, the question, here is: is it really bad to use the same term of HTTP Long Polling for the Observer? To what extent is this confusing? And, if it is, what would be the solution? Using another term (misleading to me) or maybe adding a disambiguation text?

Reply:
> One "polls" a server when software on a client periodically asks the server for something. One "pulls" data from a database towards client software.

Here we are not asking the server if there is new data, we are pulling the data no matter what every x seconds.

And yes I don't want the users to be confused with Long Polling feature

Reply:
Ok, I still have some perplexity, but "pull" works well